# FlatState
#
# Omar Elmofty 2020-03-10 -- For use in AER1217 Winter 2017
#
# Data communicates the flat state of the quadrotor
# following order [x, y, z, x_dot, y_dot, z_dot, x_ddot, y_ddot, z_ddot, yaw]

float64[9] flat_state
